organization/src/main/scala/com/improving/app/organization/domain/OrganizationValidation.scala

```diff
@@ -18,13 +18,19 @@ object OrganizationValidation {
     }
   }
 
-  val completeOrganizationInfoValidator: Validator[OrganizationInfo] =
+  val inactiveStateOrganizationInfoValidator: Validator[OrganizationInfo] =
     applyAllValidators[OrganizationInfo](Seq(
       orgInfo => organizationNameValidator(orgInfo.name),
       orgInfo => organizationNameValidator(orgInfo.shortName),
       orgInfo => required("tenant", tenantIdValidator)(orgInfo.tenant),
-      orgInfo => required("address", addressValidator)(orgInfo.address),
+      orgInfo => optional(addressValidator)(orgInfo.address),
       orgInfo => urlValidator(orgInfo.url),
       orgInfo => urlValidator(orgInfo.logo)
     ))
+
+  val activeStateOrganizationInfoValidator: Validator[OrganizationInfo] =
+    applyAllValidators[OrganizationInfo](Seq(
+      inactiveStateOrganizationInfoValidator,
+      orgInfo => required("address", addressValidator)(orgInfo.address),
+    ))
```

Comment:
Not sure why address is only being validated for active state. If this is because of how the RIDDL was written, it was probably badly written in this spot

Reply:
@AlexWeinstein92  - A functional approach is being taken here by Dag. applyAllValidtors is using inactiveStateOrganizationInfoValidator to apply 6 validations, one of which is that the address field is optional. This  active statevalidator definition is overriding address being optional and making it required.

Reply:
Makes sense - should have dug deeper into `applyAllValidators`